scripts/format.sh

```diff
@@ -6,5 +6,5 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd $DIR/../
 
 set -x
-isort --indent=4 -rc setup.py xain_fl
-black --line-length 100 setup.py xain_fl
+isort --indent=4 -rc setup.py xain_fl tests
```

Comment:
There is one small thing missing. We also need to enable it in the CI:
* `.github/workflows/development.yml`
* `.github/workflows/master.yml`

Reply:
thanks, done!